frontend/src/components/loginForm.tsx

```diff
@@ -20,7 +31,7 @@ export const LoginForm = (): JSX.Element => {
       }, {
         withCredentials: true
       })
-      navigate('/tempLandingPage')
+      window.location.href = '/hub';
```

Comment:
should this be using navigate?

Reply:
I tried using the navigate hook but it didn't reload and update the navbar. I used window.location.href to make browser do a full reload so the navbar can update based on user logged in state